src/web/demo.rkt

```diff
@@ -235,7 +235,11 @@
                   #"OK"
                   (current-seconds)
                   APPLICATION/JSON-MIME-TYPE
-                  (list (header #"Access-Control-Allow-Origin" (string->bytes/utf-8 "*")))
+                  (if (hash-has-key? resp 'job-id)
+                    (list 
+                      (header #"Access-Control-Allow-Origin" (string->bytes/utf-8 "*"))
+                      (header #"Herbie-job-id" (string->bytes/utf-8 (hash-ref resp 'job-id))))
+                    (list (header #"Access-Control-Allow-Origin" (string->bytes/utf-8 "*"))))
```

Comment:
```suggestion
                  (filter values
                    (list 
                      (header #"Access-Control-Allow-Origin" (string->bytes/utf-8 "*"))
                      (and (hash-has-key? resp 'job-id) (header #"Herbie-job-id" (string->bytes/utf-8 (hash-ref resp 'job-id))))))
```

This `filter` trick means we don't need to duplicate the `A-C-A-O` header

Reply:
Thank you!